docs/api-reference/widgets/overview.md

```diff
@@ -4,10 +4,21 @@ Widgets are UI components around the WebGL2/WebGPU canvas to offer controls and
 
 This module contains the following widgets:
 
-- [FullscreenWidget](./fullscreen-widget.md)
+### Navigation Widgets
 - [ZoomWidget](./zoom-widget.md)
+- [ResetViewWidget](./reset-view-widget.md)
```

Comment:
This is great, starting with the catalog of widgets. 

I think adding a small screenshot  of each widget here would make this even more compelling. 

I could see this being a table, with screenshot, name, short  description...

Reply:
I'd like to avoid a collection of manual screenshots since they are tedious to make and update. That was my motivation for live examples, in fact!